src/core/Table/actionHandlers/index.ts

```diff
@@ -5,9 +5,4 @@
 export { onExpandHandler } from './expandHandler';
 export { onFilterHandler } from './filterHandler';
 export { onSelectHandler, onSingleSelectHandler } from './selectHandler';
-export {
-  onTableResizeStart,
-  onTableResizeEnd,
-  TABLE_RESIZE_START_ACTION,
-  TABLE_RESIZE_END_ACTION,
```

Comment:
Why reset these?

Reply:
https://github.com/iTwin/iTwinUI-react/pull/564#discussion_r842031236

I didn't see the point of exporting them from resizeHandler if they weren't being used in there. Why not just export from Table.tsx?

Reply:
Ok, minor, still need to unify our actions.